Source/Core/VideoBackends/OGL/Render.h

```diff
@@ -36,6 +36,13 @@ enum class ES_TEXBUF_TYPE
   TEXBUF_EXT
 };
 
+enum class ES_FB_FETCH_TYPE
```

Comment:
I believe we use CamelCase for naming enum classes, please correct me if I'm wrong here.